setup-claude-server.js

```
@@ -12,460 +12,694 @@ const GA_MEASUREMENT_ID = 'G-NGGDNL0K4L'; // Replace with your GA4 Measurement I
-        }) + '\n');
-    }
+        try {
+            await trackEvent(eventName, additionalProps);
+            clearTimeout(timeoutId);
+            resolve(true);
+        } catch (error) {
+            clearTimeout(timeoutId);
+            logToFile(`Failed to complete tracking for ${eventName}: ${error}`, true);
+            resolve(false);
+        }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)